meeting_minutes/2025/2025-01-29.md

```diff
@@ -0,0 +1,72 @@
+![image](https://user-images.githubusercontent.com/1690898/139102180-5c1e2583-14f1-4f58-ab2b-9e3807ed529c.png)
+
+# Common Security Advisory Framework (CSAF) Technical Committee Working Meeting
+
+- Meeting Date: January 29, 2025
+- Time: 18:00 UTC (19:00 CET, 13:00 EST, 10:00 PST)
+
+## Call to Order and Welcome
+
+Meeting called to order @ 17:04 UTC
+
+## Roll call
+
+Inability to register attendees due to OASIS system challenges.
+
+## Participants
+
+| Given Name | Family Name | Affiliation                                                 | Role          |
+|:-----------|:------------|:------------------------------------------------------------|:--------------|
+| Christian  | Banse       | Fraunhofer-Gesellschaft e.V. - Fraunhofer AISEC             | Voting Member |
+| Christoph  | Plutte      | Ericsson AB                                                 | Voting Member |
+| Denny      | Page        | Individual                                                  | Voting Member |
+| Dina       | Truxius     | Federal Office for Information Security (BSI)               | Voting Member |
+| Feng       | Cao         | Oracle                                                      | Voting Member |
+| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Voting Member |
+| Martin     | Prpic       | Red Hat                                                     | Voting Member |
```

Comment:
According to my notes, Michael Reeder was present (but didn't had voting rights during the meeting).
```suggestion
| Martin     | Prpic       | Red Hat                                                     | Voting Member |
| Michael   | Reader   | Dell                                                             | Member              |